### DISCUSSION TITLE
how to fill the new prompts section?

### DISCUSSION THREAD

**Discussion by popedead** [d8ahazard/sd_dreambooth_extension#292] [Q&A]
Hello, I've gotten a little lost with what's new, regarding cells to fill in, in the prompts section, could someone enlighten me with an example of people and styles? Thank you. 
![dreambooth](https://user-images.githubusercontent.com/19626939/203188255-fcd1d5b7-e1b8-4e0e-b4d9-55b3a3e33788.png)

**Comment by wktra** [comment]
Changes without explanations are pretty disheartening and overwhelming. I saw that change and I just felt like giving up. I wish these changes came with more basic explanations....

**Reply by d8ahazard** [reply]
Sorry, a lot of these came about as a result of incorporating new features from a command-line only format and making them presentable to people.

As such, I do try to keep the examples up-to-date and *most* UI elements should have hints when you hover over their labels - but still, it's difficult to try adding new features while fixing bugs and addressing issues and questions while also trying to keep the docs updated.

If you have questions, try tagging me with the @ symbol, and I'll get a notification that I was mentioned. ;)

**Comment by cyperium** [comment]
The basic prompts that need to be filled in are Instance prompt and Class prompt, where Instance prompt are your "keyword", like skz or whatever they tell you to use. Class prompt is what kind of thing it is, could be "a person", "a face", etc.

I left the rest blank.
